src/components/TopBar/TopBar.js

```diff
@@ -12,7 +13,7 @@ const TopBarWrapper = styled.div`
 export const TopBar = ({ children }) => {
   return (
     <TopBarWrapper>
-      <AppBar style={{ background: '#272727' }}>
+      <AppBar style={{ background: colors.elevationOverlay04dp }}>
```

Comment:
We already have the background color for `Appbar` defined in [theme.js](https://github.com/DarthVanger/NeuralNotes/blob/master/src/theme.js#L64):
```
  overrides: {
    MuiAppBar: {
      colorPrimary: {
        backgroundColor: null,
        background: colors.elevationOverlay04dp,
      },
    },
 ```
 
 Maybe we can just delete the style here and it will use the color from the theme?

Reply:
Yes, it use the color from the theme.